src/zc/buildout/easy_install.py

```diff
@@ -37,6 +37,12 @@
 import zc.buildout
 import warnings
 
+try:
+    from setuptools.wheel import Wheel
+    SETUPTOOLS_SUPPORTS_WHEELS = True
+except ImportError:
+    SETUPTOOLS_SUPPORTS_WHEELS = False
```

Comment:
Since v38.2.0 does not support namespace packages correctly, maybe it would be best to check the version of setuptools?
```python
from pkg_resources import SetuptoolsVersion
from setuptools import __version__ as setuptools_version
SETUPTOOLS_SUPPORTS_WHEELS = SetuptoolsVersion(setuptools_version) >= SetuptoolsVersion('38.2.3')
```

Reply:
I added your version check *inside* the try/except in 5a79e918c73830cc3994402cc683c21bee334731 .
I cannot do that on the top level as there are even checks there if setuptools is even installed.